config/environments/public/public.node.config.toml

```diff
@@ -13,7 +13,7 @@ EnableLog = false
 MaxConns = 200
 
 [Pool]
-FreeClaimGasLimit = 150000
+FreeClaimGasLimit = 1500000
```

Comment:
any reason for the increment?

Reply:
Yes, due to some smc updates, right now the claim can be use more gas during an erc20 bridge where the erc20 doesn't exist yet on l2 and the contract needs to be deployed